app/inject/ShippingMethodProvider.java

```diff
@@ -0,0 +1,35 @@
+package inject;
+
+import com.google.inject.Provider;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.shippingmethods.ShippingMethod;
+import io.sphere.sdk.shippingmethods.queries.ShippingMethodQuery;
+import productcatalog.models.ShippingMethods;
+import play.Logger;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+
+@Singleton
+public class ShippingMethodProvider implements Provider<ShippingMethods> {
+    private final SphereClient client;
+
+    @Inject
+    private ShippingMethodProvider(final SphereClient client) {
+        this.client = client;
+    }
+
+    @Override
+    public ShippingMethods get() {
```

Comment:
@peter-gerhard I forgot why you needed to keep the shipping methods locally, but I just remembered we have a nice functionality on the Shipping Methods endpoint to find the appropriate shipping for a cart: http://dev.sphere.io/http-api-projects-shippingMethods.html#shipping-methods-for-cart
I think using that convenient function in Sunrise would be a nice thing to have. So please, refresh my memory :P